recipes/bigdft/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "bigdft-suite" %}
+{% set version = "1.9.3_rc.2" %}
+{% set build = 0 %}
+{% set mpi = mpi %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/l_sim/bigdft-suite/-/archive/{{ version }}/bigdft-suite-{{ version }}.tar.gz
+  md5: a756c1673c45e7e7b7be3cdf6aef81b1
+  patches:
+    - patches/output.patch
+    - patches/utils.patch
+
+build:
+  run_exports:
+     -  {{ pin_subpackage(name|lower) }}
```

Comment:
I see. I didn't understand that point at the time. I think it would be difficult to guarantee that with the current release cycle, so I've updated it to include the minor version as well.